### ISSUE TITLE
Strict Mode error in IE10

### ISSUE THREAD

**Issue by Boolian** [cgiffard/Captionator#32]
Finding IE 10 is throwing the following error:

SCRIPT5045: Assignment to read-only properties is not allowed in strict mode 
captionator.js, line 920 character 6

That particular error message is taken from your demo page.

**Comment by cgiffard** [comment]
Cheers, I'll look into it. :)

A lot has changed in the spec since I last touched Captionator, actually. I'm planning a big rewrite. If this error isn't relatively simple to fix, it'll fall into the rewrite time bracket, which could be a little while away (about a month or so.)

Sorry - but still, thanks for reporting the issue!

**Comment by Boolian** [comment]
No worries :)

Great work by the way; saved my bacon on a project I was working on!

**Comment by SureshKesana** [comment]
Any update on this?

**Comment by cgiffard** [comment]
Yep, I'm guilty of letting this languish — but I've got a rewrite on the books, using a PEG parser for VTT instead. Is it a show stopper for you right now?

**Comment by ArekvanSchaijk** [comment]
I'm running also in this problem. Does anyone has already a fix for this one?
